CMake/mlpack-config.cmake.in

```diff
@@ -0,0 +1,7 @@
+@PACKAGE_INIT@
+
+include(${CMAKE_CURRENT_LIST_DIR}/@targets_export_name@.cmake)
+
+set_and_check(MLPACK_INCLUDE_DIRS "@PACKAGE_MLPACK_INCS_DIR@")
+
+set(MLPACK_LIBRARIES mlpack::mlpack)
```

Comment:
Hey, Could you add an empty line here as well. Sorry, tiny style fix but that should re-trigger the build. :)